src/context-selection/menu-select/menu-select.js

```diff
@@ -9,7 +9,7 @@ export default function MenuSelect({ values, selected, onChange }) {
             <Menu>
                 {values.map(({ value, label }) => (
                     <MenuItem
-                        key={value}
+                        key={value || label}
```

Comment:
Wondered if we should add an optional `key` -property to the `values`-object, but this was less code...